test/QuarkBuilder.t.sol

```diff
@@ -12,6 +12,8 @@ import {Accounts} from "../src/builder/Accounts.sol";
 import {CodeJarHelper} from "../src/builder/CodeJarHelper.sol";
 import {QuarkBuilder} from "../src/builder/QuarkBuilder.sol";
 
+import "./NewFile.sol";
```

Comment:
Doesn't this need to be "test/NewFile.sol" to test the remapping?

Reply:
I believe both work correctly; just pushed `test/NewFile.sol` to confirm